test/regress/twowaysql

```diff
@@ -282,6 +282,7 @@ Racc_arg = [
   racc_shift_n,
   racc_reduce_n,
   racc_use_result_var ]
+Ractor.make_shareable(Racc_arg) if defined?(Ractor)
 
 Racc_token_to_s_table = [
   "$end",
```

Comment:
`# shareable_constant_value: literal` doesn't help?

Reply:
Ah, I forgot the magic comment, but I suppose `Ractor.make_shareable` is more appropriate than the comment.

The generated code includes user-written code, so `# shareable_constant_value: literal` will change the user-written literals. It may introduce breaking changes unexpectedly.

`Racc_` constants are Racc internal constants so we can freeze them safely.

Reply:
I see. BTW it can be off only for auto-generated code.

```ruby
# shareable_constant_value: literal
...generated code...
# shareable_constant_value: none
...user code...
```

Reply:
Cool :+1:
I tried the magic comment but unfortunately I had two problems with it so we couldn't use the magic comment :cry: 

First, a constant does not consist of literals. `Racc_arg` contains lvars. It raises Ractor::IsolationError on the const assignment.

Second, the generated code is sandwiched by user code.

```ruby
...user code (inner)...
...generated code...
...user code (actions)...
```

If the `inner` has the magic comment, the behavior will be changed by generated magic comment. For example:


```ruby
...user code (inner)...
# shareable_constant_value: literal

# shareable_constant_value: literal
...generated code...
# shareable_constant_value: none

...user code (actions)...
# The user expects `shareable_constant_value: literal` here, but doesn't.
```


I guess we can solve the first problem with `experimental_everything` but it is experimental, and we don't have any way to solve the second problem. So I think `Ractor.make_shareable` is reasonable at least for now. WDYT?